portfolio/src/main/webapp/script.js

```diff
@@ -36,4 +36,17 @@ function addRandomIntro() {
   // Add it to the page.
   const introContainer = document.getElementById('intro-container');
   introContainer.innerText = intro;
+}
+
+async function getRandomQuoteUsingAsyncAwait() {
+  const response = await fetch('/data');
+  const quote = await response.text();
+  document.getElementById('quote-container').innerText = quote;
+}
+
+function loadTasks() {
+    let list = fetch('/list-tasks').then(response => response.json());
```

Comment:
Could you change this function to async and use the await structure like you do above?